### PR TITLE
Create v2 packet class

### PR DESCRIPTION
**Pull request by peter-madigan**
Implements a new v2 packet class that is 64 bits.
Standardizes the getters + setters to use uints (instead of a mix of bitarrays / uints).

Still need to make sure this works with the hdf5 format.

### CODE REVIEW

File: larpix/format/hdf5format.py [hunk 1]
```diff
@@ -57,6 +57,118 @@
...
+          packets (``type`` == 4), and as the message timestamp in
+          message packets (``type`` == 5).
+
+        - ``dataword`` (``u1``/unsigned byte): the ADC data word
+
+        - ``trigger_type`` (``u1``/unsigned byte): the trigger type assciated
+          with this packet
+
+                ('counter','u4'),
+                ('fifo_diagnostics_enabled','u1'),
```
Comment (samkohn) [inline larpix/format/hdf5format.py]: These 2 lines look out of place?

File: larpix/format/hdf5format.py [hunk 2]
```diff
@@ -57,6 +57,118 @@
...
+        - ``chip_id`` (``u1``/unsigned byte): the LArPix chip id
+
+        - ``parity`` (``u1``/unsigned byte): the packet parity bit (0 or
+          1)
+
+        - ``valid_parity`` (``u1``/unsigned byte): 1 if the packet
+          parity is valid (odd), 0 if it is invalid
+
+        - ``downstream_marker`` (``u1``/unsigned byte): a marker to indicate the
+          hydra io network direction for this packet
```
Comment (samkohn) [inline larpix/format/hdf5format.py]: Suggested name: ``hydra_direction``

Comment (samkohn) [reply larpix/format/hdf5format.py]: Unless ``downstream_marker`` is the name used in the datasheet

Comment (peter-madigan) [reply larpix/format/hdf5format.py]: downstream marker is what is used in the datasheet, so I think we should stick with that.

File: larpix/format/hdf5format.py (hunk 1 above)
Comment (peter-madigan) [reply larpix/format/hdf5format.py]: Yes not sure what happened there, must be a copy / paste issue.

**Pull request merged**